packages/babel-plugin-redux-saga/test/index.js

```diff
@@ -96,7 +87,14 @@ utilTests.forEach(function(config) {
     var sourcePath = path.join(__dirname, 'fixtures', config.fixture, 'source.js')
     var sourceMapPath = path.join(__dirname, 'fixtures', config.fixture, 'source.js.map')
 
-    var actual = processFile(sourcePath, sourceMapPath, config.options || {}, config.pluginOptions || {})
+    var inputSourceMap = fs.existsSync(sourceMapPath)
+      ? JSON.parse(fs.readFileSync(sourceMapPath).toString())
+      : undefined
+
+    var sourceCode = fs.readFileSync(sourcePath).toString()
+    const testCaseName = path.join(config.fixture, 'source.js')
```

Comment:
u are mixing `const` and `var` here, minor nit, dont have to be changed right away

Reply:
good catch. I wanted to support old node engines so avoided using es6+ syntax (redux-saga package.json doesn't contain any requirements for node version)